README.md

```diff
@@ -169,6 +169,14 @@ This will create a folder, `<my-application-name>.chart`, in the current directo
 
 _Note that this requires the Compose Kubernetes controller available in Docker for Windows and Docker for Mac, and in Docker Enterprise Edition._
 
+### Helm chart for Docker UCP 2.0
+
+In order to create a helm chart that is compatible with version 2.0 of UCP, you will need to use the `--stack-version` flag to create a compatible version of the helm chart using `v1beta1` like so:
```

Comment:
It's actually version 3.0 of UCP which is part of the EE 2.0 platform :)

Reply:
Let's just say EE 2.0

Reply:
Plot twist! :p

Fixed now 👍